plugins/coral-plugin-offtopic/client/containers/OffTopicCheckbox.js

```diff
@@ -0,0 +1,11 @@
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {addTag, removeTag} from 'coral-plugin-commentbox/actions';
+import OffTopicCheckbox from '../components/OffTopicCheckbox';
+
+const mapStateToProps = ({commentBox}) => ({commentBox});
```

Comment:
This would ensure future evolution of talk, were we want to have multiple embeds with connected store, and some store restructuring will be required.